SAGA/spectra/read_external.py

```diff
@@ -130,6 +132,33 @@ def read_ozdes(file_path):
     return ensure_specs_dtype(specs)
 
 
+def read_wigglez(file_path):
+    if not hasattr(file_path, 'read'):
+        file_path = FitsTable(file_path)
+
+     specs = file_path.read()['RAJ2000', 'DEJ2000', 'Name', 'z', 'q_z']
+
+    # 3 = probably galaxy, 4 = definite galaxy, 6 = confirmed star
+    specs = Query('q_z >= 3').filter(specs)
+
+    specs.rename_column('WIGGLEZ', 'SPECOBJID')
```

Comment:
`"WIGGLEZ"` is not a column of `specs` --- unless Line 139 is missing `"WIGGLEZ"` in it?

```python 
specs = file_path.read()['RAJ2000', 'DEJ2000', 'Name', 'z', 'q_z']
```